### PR TITLE
Elecfreaks picoed

### PR DESCRIPTION
**Pull request by xuhow**
Request to add ELECFREAKS Pico:ed board.

### CODE REVIEW

File: ports/raspberrypi/boards/elecfreaks_picoed/pins.c [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    { MP_ROM_QSTR(MP_QSTR_P10), MP_ROM_PTR(&pin_GPIO10) },
+    { MP_ROM_QSTR(MP_QSTR_P11), MP_ROM_PTR(&pin_GPIO11) },
+    { MP_ROM_QSTR(MP_QSTR_P12), MP_ROM_PTR(&pin_GPIO12) },
+    { MP_ROM_QSTR(MP_QSTR_P13), MP_ROM_PTR(&pin_GPIO13) },
+    { MP_ROM_QSTR(MP_QSTR_P14), MP_ROM_PTR(&pin_GPIO14) },
+    { MP_ROM_QSTR(MP_QSTR_P15), MP_ROM_PTR(&pin_GPIO15) },
+    { MP_ROM_QSTR(MP_QSTR_P16), MP_ROM_PTR(&pin_GPIO16) },
+
+    { MP_ROM_QSTR(MP_QSTR_SDA), MP_ROM_PTR(&pin_GPIO18) },
+    { MP_ROM_QSTR(MP_QSTR_P20), MP_ROM_PTR(&pin_GPIO18) },
```
Comment (gamblor21) [inline ports/raspberrypi/boards/elecfreaks_picoed/pins.c:29]: Question did you mean to have P20 map to GPIO18? Everything else seems to match so just wanted to make sure this was on purpose.

**Review by gamblor21 — COMMENTED** [review]
One comment just to make sure the mapping was as intended. The rest looks good.

File: ports/raspberrypi/boards/elecfreaks_picoed/pins.c (hunk 1 above)
Comment (xuhow) [reply ports/raspberrypi/boards/elecfreaks_picoed/pins.c:29]: Yes. This is to keep the pins of Pico:ed compatible with the micro:bit edge connector.

Comment (gamblor21) [reply ports/raspberrypi/boards/elecfreaks_picoed/pins.c:29]: Sounds good!

**Review by gamblor21 — APPROVED** [review]
Looks all good to me. Could not test (as I do not have the hardware obviously).

**Comment by gamblor21** [comment]
@dhalbert @jepler The board definition looks fine but as this is a new board just wanted to check if there are any other steps to be taken before it is merged in?

**Review by dhalbert — APPROVED** [review]
This looks good to me. @xuhow, could you now also prepare a PR for https://github.com/adafruit/circuitpython-org, to add a description and photos?

**Comment by xuhow** [comment]
> This looks good to me. @xuhow, could you now also prepare a PR for https://github.com/adafruit/circuitpython-org, to add a description and photos?

Yes, I just submitted this [PR](https://github.com/adafruit/circuitpython-org/pull/934)

**Pull request merged**